source/stylist/style.py

```diff
@@ -45,7 +45,11 @@ def check(self,
         logging.getLogger(__name__).info('Style: ' + self.__class__.__name__)
         issues: List[stylist.issue.Issue] = []
         for rule in self._rules:
-            issues.extend(rule.examine(source))
+            additions = rule.examine(source)
```

Comment:
`additions` seems a bit ambiguous a name here `additional_issues` is perhaps better to make it clear you're adding issues to the over all list.

Reply:
And done.